usaspending_api/references/v2/views/cfda.py

```diff
@@ -0,0 +1,48 @@
+from rest_framework.response import Response
+from rest_framework.views import APIView
+from requests import post
+from time import sleep
+from django.conf import settings
+
+ALL_CFDAS = None
+
+
+class CFDAViewSet(APIView):
+    """
+    Return an agency name and active fy.
+    """
+
+    endpoint_doc = "usaspending_api/api_contracts/contracts/v2/references/agency/id.md"
```

Comment:
endpoint doc string is wrong

Reply:
This was probably updated while you were reviewing.